src/content/en/2021/caching.md

```diff
@@ -17,4 +19,214 @@ featured_stat_3: TODO
 featured_stat_label_3: TODO
 ---
 
-## TODO
+## Introduction
+
+Caching is a technique that enables the reuse of previously downloaded content. Over the past decade...
+
+Why do we cache? What's the benefit?
+
+The three main pillars of caching web content are:
+- Cache as much as you can
+- Cache for as long as you can
+- Cache as close to end users as you can
+
+
+## CDN Cache adoption
+
+Talk about CDN adoptions, 14% of desktop pages use a Cloudflare CDN.
+
```

Comment:
A Content Delivery Network (CDN) is a group of servers spread out over several locations that store copies of data. This allows servers to fulfill requests based on the server closest to the end-user. (Remember we want to cache as close to end-users as possible.) Across the web, 14% of Desktop and Mobile pages use a CDN. The most popular is Cloudflare, followed by Google at 7% adoption for Mobile. While Cloudflare is used twice as much as Google, there remains a large variety of solutions available.